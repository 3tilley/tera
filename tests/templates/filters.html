<html>
  <head>
    <title>{{ product.name | upper | trim }}</title>
  </head>
  <body>
    <h1>{{ product.name | trim }} - {{ product.manufacturer | trim | truncate(length=4) }}</h1>
  </body>
</html>
