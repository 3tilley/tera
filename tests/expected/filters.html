<html>
  <head>
    <title>MOTO G</title>
  </head>
  <body>
    <h1>Moto G - Motoâ€¦</h1>
  </body>
</html>
